src/libtriton/arch/arm/aarch64/aarch64Cpu.cpp

```diff
@@ -297,21 +297,29 @@ namespace triton {
             auto regId = kv.first;
             const auto& reg = kv.second;
 
-            /* Skip Vector and System registers */
-            if (this->isVectorRegister(regId) || this->isSystemRegister(regId))
```

Comment:
Why were the system registers being skipped originally? Is there a reason they should not be included?

Reply:
If i remember correctly it was because I did not want to pollute to much the output. So, clearly not a good reason ^^.